tests/Support/CacheStoreTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Spatie\Ray\Tests\Support;
+
+use PHPUnit\Framework\TestCase;
+use Spatie\Ray\Support\CacheStore;
+use Spatie\Ray\Tests\TestClasses\ClockMock;
+
+class CacheStoreTest extends TestCase
+{
+    /** @var ClockMock */
+    private $clock;
```

Comment:
Why `private` and not `protected` like in the rest of the package?

Reply:
Force of habit. Will change